### ISSUE TITLE
Giving an indicator whether a known site must be selected vs a new site can be created for an address

### ISSUE THREAD

**Issue by fahimsabir-colt** [MEF-GIT/MEF-LSO-Sonata-SDK#28]
With certain addresses, a carrier may opt only to deliver to sites which have already been established, not allowing any new sites to be created.  An indicator on the Address Validation API of this (against each address) would be very helpful to inform the buyer and allow them to understand a qualification, quote, or order will get rejected before they submit it.

An example of where this happens is in datacentres - where the carrier only wants to use MMRs (Meet Me Rooms) rather than cabling to racks (which is an option, but not one the carrier wants to use).

**Comment by MichalLaczynskiAmartus** [comment]
Is allowNewSites enough?

**Comment by fahimsabir-colt** [comment]
Yes. this comment was raised before the allowNewSites attribute was added.
